unstructured/metrics/element_type.py

```diff
@@ -0,0 +1,19 @@
+from typing import Dict, List, Union
+
+
+def get_element_type_frequency(
+    elements: List,
+) -> Union[Dict[str, Dict[str, int]], Dict]:
+    frequency: Dict = {}
```

Comment:
Can we have this dictionary initialize all element categories to 0 (like a default dict)? Will help with comparison between 2 such dictionaries since they will both have the same keys